docusaurus.config.js

```diff
@@ -93,52 +94,48 @@ module.exports = {
         {
           title: 'Developers',
           items: [
+            {
+              label: 'Dev Chat',
+              href: 'https://discord.com/invite/uniswap',
+            },
             {
               label: 'Feedback',
               href: 'https://forms.gle/13XtjmkwdXQ2jMn26',
             },
             {
               label: 'Bug Bounty',
-              href: 'https://github.com/Uniswap/uniswap-v3-periphery/blob/main/bug-bounty.md',
-            },
-            {
-              label: '#dev-chat',
-              href: 'https://discord.com/invite/uniswap',
+              href: 'https://blog.uniswap.org/v4-bug-bounty',
             },
             {
               label: 'Whitepaper',
-              href: 'https://uniswap.org/whitepaper-v3.pdf',
+              href: 'https://app.uniswap.org/whitepaper-v4.pdf',
             },
           ],
         },
         {
           title: 'Github',
           items: [
             {
-              label: 'uniswap-v3-core',
-              href: 'https://github.com/Uniswap/uniswap-v3-core',
+              label: 'uniswap-v4-core',
+              href: 'https://github.com/Uniswap/v4-core',
             },
             {
-              label: 'uniswap-v3-sdk',
-              href: 'https://github.com/Uniswap/uniswap-v3-sdk',
+              label: 'uniswap-v4-sdk',
+              href: 'https://github.com/Uniswap/sdks/tree/main/sdks/v4-sdk',
             },
             {
-              label: 'uniswap-v3-periphery',
-              href: 'https://github.com/Uniswap/uniswap-v3-periphery',
+              label: 'uniswap-v4-periphery',
+              href: 'https://github.com/Uniswap/v4-periphery',
             },
             {
               label: 'Deployment addresses',
-              href: 'https://github.com/Uniswap/uniswap-v3-periphery/blob/main/deploys.md',
+              href: 'https://docs.uniswap.org/contracts/v4/deployments',
```

Comment:
we should avoid hard-linking to the domain as it might change
```suggestion
              href: '/contracts/v4/deployments',
```